zipkin-cassandra-core/src/main/java/org/twitter/zipkin/storage/cassandra/Repository.java

```diff
@@ -205,6 +220,28 @@ public Repository(String keyspace, Cluster cluster, Boolean ensureSchema) {
                     .value("ts", QueryBuilder.bindMarker("ts"))
                     .value("trace_id", QueryBuilder.bindMarker("trace_id"))
                     .using(QueryBuilder.ttl(QueryBuilder.bindMarker("ttl_"))));
+
+        selectTraceIdsBySpanDuration = session.prepare(
+                QueryBuilder.select("d", "ts", "tid")
+                    .from("span_duration_index")
+                    .where(QueryBuilder.eq("s", QueryBuilder.bindMarker("service_name")))
```

Comment:
Yes, this was with compression. The schema is like below. Only the n(name) field would have longer strings, others are usually a single number, which become comparable in space usage to column names if those are fully spelled out.

The x5 size difference may not be directly attributed to column names, but that's the best measure they had. I don't know how much of intellectual burden short names really are, there are like 10 lines of code that depend on them. Taking extra disk space affects performance in various ways, in streaming, further compactions, etc.

But I am not strongly against changing this to long names.
```
CREATE TABLE events (
  n text,
  rt bigint,
  t bigint,
  v blob,
  PRIMARY KEY ((n, rt), t)
) WITH compression={'sstable_compression': 'LZ4Compressor'};
```